src/xcode/ENA/ENA/Resources/Localization/ar.lproj/Localizable.strings

```diff
@@ -0,0 +1,900 @@
+
+"OnboardingInfo_enableLoggingOfContactsPage_panelTitle" = "إقرار الموافقة";
+
+"OnboardingInfo_enableLoggingOfContactsPage_panelBody" = "لمعرفة ما إذا كنت قد تواصلت مع شخص مصاب وما إذا كان هناك خطر في تعرضك للإصابة، يجب عليك تمكين ميزة تسجيل المخالطة في التطبيق. من خلال النقر فوق زر \"تشغيل\"، توافق على تمكين ميزة تسجيل المخالطة في التطبيق وما يرتبط بها من معالجة للبيانات.\n\n\nلاستخدام ميزة تسجيل المخالطة في التطبيق، سيتعين عليك تمكين وظيفة تسجيل المخالطة لمرض كوفيد-19 المقدمة من شركة Apple على هاتف iPhone لديك ومنح تطبيق Corona-Warn-App الإذن باستخدام هذه الميزة.\n\nعند تمكين تسجيل المخالطة، ينشئ هاتف iPhone المعرفات العشوائية ويرسلها عبر Bluetooth باستمرار، ويمكن لهواتف Apple وAndroid الذكية الأخرى الموجودة في منطقتك تلقيها إذا تم تمكين تسجيل المخالطة عليها أيضًا. وبدوره، يتلقى هاتف iPhone لديك المعرفات العشوائية للهواتف الذكية الأخرى، ويتم تسجيل معرفاتك العشوائية وتلك المتلقاة من الهواتف الذكية الأخرى في سجل المخالطة وتخزينها فيه لمدة 14 يومًا.\n\n\nلمعرفة خطر العدوى عليك، يقوم التطبيق، عدة مرات في اليوم أو حسب الطلب، بتحميل قائمة بالمعرفات العشوائية لكل المستخدمين الذين أخطروا التطبيق بأنهم مصابون بفيروس كورونا. وتتم مقارنة هذه القائمة، بعد ذلك، بالمعرفات العشوائية المخزنة في سجل المخالطة. في حالة اكتشاف التطبيق أنك قد تواصلت مع مستخدم مصاب، فسيخطرك بذلك ويخبرك بوجود خطر تعرضك للإصابة أيضًا. في هذه الحالة، يتم أيضًا منح التطبيق حق الوصول إلى البيانات الأخرى المسجلة في سجل المخالطة بهاتف iPhone لديك (تاريخ التواصل ومدته وقوة إشارة Bluetooth الخاصة به).\n\nتُستخدم قوة إشارة Bluetooth لاستنتاج بُعد المسافة (كلما زادت قوة الإشارة، كانت المسافة أقصر). وبعد ذلك، يحلل التطبيق هذه المعلومات من أجل تقييم احتمالية إصابتك بفيروس كورونا ولتقديم التوصيات إليك بشأن ما يجب فعله بعد ذلك. يتم إجراء هذا التحليل فقط محليًا على هاتف iPhone لديك. وبخلافك أنت، لن يعرف أي شخص (ولا حتى معهد روبرت كوخ) ما إذا كنت قد تواصلت مع شخص مصاب والخطر الذي تم تحديده لك.\n\nلسحب موافقتك على استخدام ميزة تسجيل المخالطة، يمكنك تعطيل الميزة باستخدام زر التبديل في التطبيق أو حذف التطبيق. وإذا قررت استخدام الميزة مرة أخرى، يمكنك تشغيل الميزة مرة أخرى أو إعادة تثبيت التطبيق. إذا قمت بتعطيل ميزة تسجيل المخالطة، فلن يتحقق التطبيق بعد ذلك مما إذا كنت قد تواصلت مع مستخدم مصاب. وإذا كنت ترغب أيضًا في إيقاف إرسال جهازك للمعرفات العشوائية وتلقيها، يتعين عليك تعطيل ميزة تسجيل المخالطة لمرض كوفيد-19 في إعدادات هاتف iPhone لديك. الرجاء ملاحظة أنه لن يتم حذف معرفاتك العشوائية وتلك المتلقاة من الهواتف الذكية الأخرى المخزنة في سجل المخالطة من التطبيق. يمكنك فقط حذف البيانات المخزنة في سجل المخالطة في إعدادات هاتف iPhone نهائيًا.\n\n\nيمكنك العثور على إشعار خصوصية التطبيق (بما فيه توضيح لمعالجة البيانات المنفَّذة لميزة تسجيل المخالطة) في القائمة ضمن \"معلومات خصوصية البيانات\".";
+
+"OnboardingInfo_enableLoggingOfContactsPage_stateHeader" = "Status";
+
+"OnboardingInfo_enableLoggingOfContactsPage_stateTitle" = "Risiko-Ermittlung";
+
+"OnboardingInfo_enableLoggingOfContactsPage_stateActive" = "Aktiv";
+
+"OnboardingInfo_enableLoggingOfContactsPage_stateStopped" = "Gestoppt";
```

Comment:
Intentionally or accidentally left untranslated?

Reply:
intentional, will be translated later on